app/assets/stylesheets/modules/map/map.css.scss

```diff
@@ -373,6 +373,11 @@ p.map-popup-info {
   height: 180px;
 }
 
+#bounding-box-map {
```

Comment:
I'd add `cursor: default;" to this selector as well. Since the map isn't able to be moved/dragged it makes sense to keep the cursor the arrow instead of how it is currently behaving.